.gitignore

```diff
@@ -18,7 +18,6 @@
 # Ignore ruby version management artifacts.
 /.ruby-*
 
-/config/gdor.yml
 /log/gdor_logs/
```

Comment:
Should these be removed as well?

Reply:
no, we've repurposed this directory for harvestdor logs